src/routers/DeleteUserRouter.ts

```diff
@@ -12,8 +12,7 @@ class DeleteUserRouter extends ApplicationRouter<void> {
   }
 
   async content(req: Request): Promise<void> {
-    const { netID } = req.query;
-    await UserRepo.deleteUser(netID);
+    await UserRepo.deleteUser(req.user);
```

Comment:
So the entire user has to be sent in the request body? If so, feel like we should create a `deleteUserByNetID` function in the UserRepo so that we don't have to send the entire user object for requests like this where it's not necessary.